web/src/components/molecules/Integration/IntegrationConnectModal/integrationCrad.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from "@emotion/styled";
+import React from "react";
+
+export interface Props {
+  src?: string;
+  selected?: boolean;
+  title: string;
+}
```

Comment:
```suggestion
export type Props = {
  src?: string;
  selected?: boolean;
  title: string;
}
```
Just a suggestion (you can ignore it) since you are using interface as a type.

Reply:
I don't know what happened to my previous comment here, but I asked for you to follow through with this and change this to type since across repos we favour type over interface

Reply:
https://github.com/reearth/reearth-cms/blob/1d640610d413befc0df76c9533bae2b216b397de/web/src/components/molecules/Integration/IntegrationConnectModal/IntegrationCard.tsx#L3